src/reference/asciidoc/amqp.adoc

```diff
@@ -2126,6 +2126,42 @@ static class TxServiceImpl implements TxService<Foo> {
 }
 ----
 
+[[annotation-error-handling]]
+====== Handling Exceptions
+
+By default, if an annotated listener method throws an exception, the exeption is thrown to the container and the message will be requeued and redelivered, discarded, or routed to a Dead Letter Exchange, depending on the container and broker configuration.
+Nothing is returned to the sender.
+
+Starting with _version 2.0_, the `@RabbitListener` annotation has two new attributes: `errorHandler` and `returnExceptions`.
+
+These are not configured by default.
+
+Use the `errorHandler` to provide the bean name of a `RabbitListenerErrorHandler` implementation.
+This functional interface has one method:
+
+[source, java]
+----
+@FunctionalInterface
+public interface RabbitListenerErrorHandler {
+
+	Object handleError(Message amqpMessage, org.springframework.messaging.Message<?> message,
+			ListenerExecutionFailedException exception) throws Exception;
```

Comment:
Whitespaces for indents instead of tabs.
Yes, in docs only 😄 

Fixing on merge...